add_qr_cta_to_books.py

```
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
+"""
+Add QR codes and CTAs to puzzle books for lead generation
+Embeds email capture mechanisms directly into generated PDFs
+"""
+
+import qrcode
+from PIL import Image, ImageDraw, ImageFont
```

Flake8 (7.2.0)

[error] 'PIL.ImageDraw' imported but unused

(F401)